build/esbuild-build.ts

```diff
@@ -53,11 +53,9 @@ function createEnvDefines(envVarNames: string[], extras: Record<string, unknown>
       throw new Error(`Missing environment variable: ${name}`);
     }
   }
-  for (const key in extras) {
-    if (Object.prototype.hasOwnProperty.call(extras, key)) {
-      defines[key] = JSON.stringify(extras[key]);
-    }
-  }
+  Object.entries(extras).forEach(([key, value]) => {
+    defines[key] = JSON.stringify(value);
+  });
   defines["extraRpcs"] = JSON.stringify(allNetworkUrls);
```

Comment:
This whole build file is outdated. Can you take the latest from pay.ubq.fi

Reply:
It is now up to date. Note that `pay.ubq.fi` is also using a `for .. in` loop, might wanna update that too